documentation/1.10/content/userguide/tools/create-aux-image.md

```diff
@@ -0,0 +1,84 @@
+| `--builder`, `-b` | Executable to process the Dockerfile. Use the full path of the executable if not on your path. | `docker`  |
+| `--buildNetwork` | Networking mode for the RUN instructions during the image build.  See `--network` for Docker `build`.  |   |
+| `--chown` | `userid:groupid` for JDK/Middleware installs and patches.  | `oracle:oracle` |
+| `--dryRun` | Skip Docker build execution and print the Dockerfile to stdout.  |  |
+| `--fromImage` | Container image to use as a base image when creating a new image. | `busybox`  |
+| `--httpProxyUrl` | Proxy for the HTTP protocol. Example: `http://myproxy:80` or `http:user:passwd@myproxy:8080`  |   |
+| `--httpsProxyUrl` | Proxy for the HTTPS protocol. Example: `https://myproxy:80` or `https:user:passwd@myproxy:8080`  |   |
+| `--packageManager` | Override the default package manager for the base image's operating system. Supported values: `APK`, `APTGET`, `NONE`, `YUM`, `ZYPPER`  |   |
+| `--pull` | Always attempt to pull a newer version of base images during the build.  |   |
+| `--tag` | (Required) Tag for the final build image. Example: `store/oracle/mydomain:1`  |   |
+| `--target` | Select the target environment in which the created image will be used. Supported values: `Default` (Docker/Kubernetes), `OpenShift` | `Default`  |
+| `--wdtArchive` | A WDT archive zip file or comma-separated list of files.  |   |
+| `--wdtHome` | The target folder in the image for the WDT install and models.  | `/auxiliary`  |
+| `--wdtModel` | A WDT model file or a comma-separated list of files.  |   |
+| `--wdtModelHome` | The target location in the image to copy WDT model, variable, and archive files. | `$WDT_HOME/models` |
+| `--wdtVariables` | A WDT variables file or comma-separated list of files.  |   |
+| `--wdtVersion` | WDT version to use.  | `latest`  |
+
+### Additional information
+
+#### `--target`
+
+By default, the generated WLS domain in your image will use the best practices defined by Oracle WebLogic Server.  
+The `target` option allows you to toggle the defaults so that the generated domain is easier to use in the target
+environment.  For example, the `--target OpenShift` option will change the file permissions in the domain directory
+so that the group permissions match the user permissions.
+
+| Target | Default File Permissions | Default File Ownership |
+| --- | --- | --- |
+| `Default` | `rwxr-x---` | `oracle:oracle` |
+| `OpenShift` | `rwxrwx---` | `oracle:root` |
+
+#### Use an argument file
+
+You can save all arguments passed for the Image Tool in a file, then use the file as a parameter.
+
+For example, create a file called `build_args`:
+
+```bash
+createAuxImage
+--tag mydomain:1
+--wdtModel ./my_domain.yaml
+--wdtArchive ./my_domain.zip
+```
+
+Use it on the command line, as follows:
+
+```bash
+$ imagetool @/path/to/build_args
+```
+
+
+### Usage scenarios
+
+The commands below assume that all the required WDT installer has been downloaded and added to the cache.
```

Comment:
The commands below -> The following commands 
assume that all the required WDT installer has been downloaded -> (this doesn't make sense to me, maybe) assume that the required WDT installer has been downloaded